### DISCUSSION TITLE
Browser Version

### DISCUSSION THREAD

**Discussion by toomanylogins** [terser/html-minifier-terser#109] [Q&A]
Is there a build of this that can be run client side in the browser
thanks

**Comment by DanielRuf** [comment]
Sure, see https://terser.org/html-minifier-terser and the relevant files at https://github.com/terser/html-minifier-terser/tree/master/demo

**Comment by sibiraj-s** [comment]
Starting v7 browser builds are distributed to npm. See https://unpkg.com/browse/html-minifier-terser@7.0.0-alpha.1/dist

<img width="912" alt="Screenshot 2022-01-30 at 1 53 32 PM" src="https://user-images.githubusercontent.com/20282546/151692294-5e5ad8d5-3f65-4dfd-af6a-3c8f118aba30.png">

Prior to v7, bundles are not distributed to npm. You can use any cdn that can import files from github. You can find the files in the `dist/` directory https://github.com/terser/html-minifier-terser/tree/v6.1.0/dist

**Comment by toomanylogins** [comment]
Thank you for reply. Apologies if this seems trivial. I dev db applications (4d.com). Running example code in a headless browser I cant get past this error in the web inspector.
VM216:3 ReferenceError: minify is not defined
    at <anonymous>:1:5
try{minify('<p title="blah" id="moo">foo</p>', {
  removeAttributeQuotes: true,
});}catch(e){console.error(e);}

Is the minify() the correct function to call with the source html as the first param ? 
Thanks

**Comment by sibiraj-s** [comment]
Minify is not added to global variables. If you are using via UMD builds, you will need to access via UMD namespace.

```html
<body>
  <script src="https://cdn.jsdelivr.net/npm/html-minifier-terser@7.0.0-alpha.1/dist/htmlminifier.umd.bundle.js"></script>
  <script>
    const minify = window.HTMLMinifier.minify;
  </script>
</body>
```

Or For modules.

```html
<script type="module">
  import HTMLMinifier from 'https://cdn.skypack.dev/html-minifier-terser';

  const minify = HTMLMinifier.minify
</script>
```

**Comment by toomanylogins** [comment]
Thank you for trying to help. This is offline desktop application. Testing with your example for UMD build
`https://cdn.jsdelivr.net/npm/html-minifier-terser@7.0.0-alpha.1/dist/htmlminifier.umd.bundle.js`
I no longer get error in the inspector however all I can get returned is `[object Object]`
